docs/md/se/jobs/11619.md

```diff
@@ -0,0 +1,35 @@
+# Agiler Software Engineer Java (m/w) 80-100%
+
+## Was dich erwartet
+
+Unser [Aufgabenspektrum](https://www.baloise.com/jobs/de/blog/baloise-jobcast/jobcast-it-engineer.html) ist klar. Der Fachbereich kommt mit einer Anforderung, wir besprechen diese gemeinsam und setzen dann entsprechend um.
+
+Klingt mega unspannend, ist es aber nicht, weil du bei uns die kreative Freiheit bekommst, eigene Lösungen zu finden. Wir sind so gar nicht top down, sondern vor allem auf Augenhöhe unterwegs. Du bist die IT-Kompetenz - vom Front- bis ins Backend, Java- bzw. Angular-basiert – und du bringst deine Vorstellung von einer möglichen Umsetzung ein.
+
+Warum das Spass macht? Weil wir nah beim Kunden sind. So nah wie man das als IT'ler vielleicht von einem Versicherer nicht erwartet. Wir bekommen direktes Feedback und können unsere Software nutzergerecht optimieren. Du bekommst auch genug Challenge im Team. Wenn du also Spass hast an Austausch, Diskussion und gemeinsamer Entwicklung bist du hier richtig. 
+
+## Was wir erwarten
+
+Dass du Lust auf Community hast. Ein grosses Plus bei der Baloise ist, dass wir uns in einem [agilen Umfeld](https://www.baloise.com/jobs/de/blog/baloise-jobcast/baloise-jobcast-3-autonome-teams-in-der-it-entwicklung.html) in immer neuen Teams zusammenfinden. Das stärkt das Netzwerk und die Zusammenarbeit. Sei neugierig, hab den Antrieb, dich weiterzubilden und nutze die Möglichkeiten. Es gibt regelmässig Fortbildungsangebote auf Konferenzen und Messen, die dich und damit die Baloise weiterbringen. Open X Days, Stages inbegriffen.
```

Comment:
We do not attend "Messen" - besides conferences I would mention meetups, courses / trainings

Reply:
What are Open X Days? Maybe mention [OpenSpace](https://en.wikipedia.org/wiki/Open_Space_Technology), what exactly is a Stage? Maybe explain it.